hastory-cli/src/Hastory/Cli/OptParse.hs

```diff
@@ -65,17 +65,18 @@ combineToInstructions cmd Flags {..} Environment {..} mConf =
         CommandRegister registerFlags -> DispatchRegister . RegisterSettings <$> getRemoteStorage registerFlags
     getRemoteStorage :: RemoteStorageFlags -> IO RemoteStorage
     getRemoteStorage RemoteStorageFlags {..} = do
+      let helpMsg = "Use 'hastory -h' or 'hastory <subcommand> -h' for usage details."
       remoteStorageBaseUrl <-
         case remoteStorageFlagsServer <|> envStorageServer <|> mc configStorageServer of
-          Nothing -> die "Storage server not found"
+          Nothing -> die $ unwords ["URL not found.", helpMsg]
```

Comment:
I'm not sure this is necessary.
People know what `--help` does. (We should be recommending `--help`, by the way, not `-h`.)

I also don't like "URL" as a description of what's not found.
We can use `Storage server not configured` instead. The same goes for the others below.

Reply:
> Looks good, please rebase onto master because I had to fix CI.

@NorfairKing Thank you for the review. I made the suggested changes:

0. Merged in `master` so that CI passes.
0. Removed the `helpMsg`.